frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -338,56 +404,152 @@ export default function AnalysisParamForm({
+        </WALabel>
+        <AdvancedAnalysisContainer>
+          <WAOverlayContainer>
+            {checkedAdvanced || <WAOverlay />}
+            <TwoParamContainer>
+              <Label htmlFor="prominenceFactorMin">Prominence (uN):</Label>
+              <InputErrorContainer>
+                <label htmlFor="prominenceFactorMin">min</label>
+                <FormModify>
+                  <FormInput
+                    name="prominenceFactorMin"
+                    placeholder={checkedAdvanced ? "0" : ""}
+                    value={!checkedAdvanced ? "" : inputVals.prominenceFactorMin}
+                    onChangeFn={(e) => {
+                      updateParams({
+                        prominenceFactorMin: e.target.value,
+                      });
+                    }}
+                  >
+                    <ErrorText id="prominenceFactorMinError" role="errorMsg">
+                      {errorMessages.prominenceFactorMin}
+                    </ErrorText>
+                  </FormInput>
+                </FormModify>
+                <label htmlFor="prominenceFactorMax">max</label>
+                <FormModify>
+                  <FormInput
+                    name="prominenceFactorMax"
+                    placeholder={checkedAdvanced ? "100" : ""}
+                    value={!checkedAdvanced ? "" : inputVals.prominenceFactorMax}
+                    onChangeFn={(e) => {
+                      updateParams({
+                        prominenceFactorMax: e.target.value,
+                      });
+                    }}
+                  >
+                    <ErrorText id="prominenceFactorMaxError" role="errorMsg">
+                      {errorMessages.prominenceFactorMax}
+                    </ErrorText>
+                  </FormInput>
+                </FormModify>
+
+              </InputErrorContainer>
+            </TwoParamContainer>
+            <TwoParamContainer>
+              <Label htmlFor="widthFactorMin">Width (ms):</Label>
+              <InputErrorContainer>
+                <label htmlFor="widthFactorMin">min</label>
+                <FormModify>
+                  <FormInput
+                    name="widthFactorMin"
+                    placeholder={checkedAdvanced ? "0" : ""}
+                    value={!checkedAdvanced ? "" : inputVals.widthFactorMin}
+                    onChangeFn={(e) => {
+                      updateParams({
+                        widthFactorMin: e.target.value,
+                      });
+                    }}
+                  >
+                    <ErrorText id="widthFactorMinError" role="errorMsg">
+                      {errorMessages.widthFactorMin}
+                    </ErrorText>
+
+                  </FormInput>
+                </FormModify>
+                <label htmlFor="widthFactorMax">max</label>
+                <FormModify>
+                  <FormInput
+                    name="widthFactorMax"
+                    placeholder={checkedAdvanced ? "100" : ""}
```

Comment:
If there's no default max in the BE/pulse3d then we'll want to use None/null here

Reply:
I think it’s defaulting to 7 for both in pulse3D and prominence is defaulting to 6 for both, but yeah might be nice to make sure it all works with None/null

Reply:
Now that I'm looking through the pulse3d code again, the two values in the parametrization aren't actually a max and min, they're factors used in determining the min value for peaks and the min value for valleys respectively. I think we'll definitely need to discuss more with Jason and Hamed tomorrow